app/src/main/java/com/smart/learning/domain/User.java

```diff
@@ -39,7 +39,7 @@
     private Set<Authority> authorities = new HashSet<>();
 
     @Email
-    @Indexed
+    @Indexed(unique = true)
```

Comment:
I think this restriction should not be there. We only get an email as valid when user verifies their email address. Before that, someone can enter someone else's email address and we don't want to prevent the real owner from registration.